app/src/main/res/values/strings.xml

```diff
@@ -103,6 +103,8 @@
 
     <!-- Pinned Messages -->
     <string name="title_pinned_messages">Pinned Messages</string>
+    <string name="no_pinned_messages">No pinned Messages</string>
```

Comment:
Could you add the Brazilian translation for this string?

No pinned Messages -> Nenhuma mensagem pinada.